ci/Dockerfile.ruby3

```diff
@@ -0,0 +1,36 @@
+#!BuildTag: rmt-ci-container-ruby3.2
+#!UseOBSRepositories
+FROM opensuse/tumbleweed
+
+RUN zypper --non-interactive install --no-recommends ruby3.2 ruby3.2-devel
+    
+RUN zypper --non-interactive install libffi-devel libmysqlclient-devel libxml2-devel \
+                        libxslt-devel rpmbuild systemd gzip tar bzip2 nodejs sqlite-devel \
+                        make chrpath fdupes gcc libcurl-devel
+
+RUN rm /usr/bin/ruby /usr/bin/gem
+
+RUN update-alternatives --install /usr/bin/gem gem /usr/bin/gem.ruby3.2 1
+RUN update-alternatives --install /usr/bin/gem gem /usr/bin/gem.ruby3.4 2
```

Comment:
do we actually need to register 3.4 for this PR?

Reply:
Not for this PR but it is needed in general as tumbleweed ships with ruby 3.4 as a default.